cashctrl_ledger/tests/test_ledger_attachments.py

```diff
@@ -71,7 +71,7 @@ def ledger_ids():
     yield ledger_ids
 
     # Restore original ledger state
-    engine.delete_ledger_entry([str(id) for id in ledger_ids])
+    engine.delete_ledger_entries([str(id) for id in ledger_ids])
```

Comment:
Let's not require the id to be a string. Please change to:
```
engine.delete_ledger_entries(ledger_ids)
```
instead of:
```
engine.delete_ledger_entries([str(id) for id in ledger_ids])
```

(Also applies to L91)